app/models/fagus/item.rb

```diff
@@ -0,0 +1,35 @@
+# Fagus
+module Fagus
+  # Represents an item.
+  class Item < ActiveRecord::Base
+    extend Enumerize
+
+    belongs_to :category
+    belongs_to :item_type
+
+    enumerize :status,
+              :in => [:active, :phased_out, :discontinued],
+              :default => :active
+
+    validates \
+      :category,
+      :description,
+      :item_type,
+      :reference,
+      :status,
```

Comment:
Interesting. That won't give us a validation error. More of a side effect of enumerize. My suggestion is to validate just to make sure something built on top of this model will behave correctly.